torch/utils/cpp_extension.py

```diff
@@ -284,25 +284,26 @@ def _join_sycl_home(*paths) -> str:
 
 _COMMON_SYCL_FLAGS = [
     '-fsycl',
-    '-fsycl-targets=spir64_gen,spir64',
```

Comment:
`-fsycl-targets=spir64_gen,spir64` is a kernel flag I remember.

Reply:
Refer to https://github.com/intel/llvm/blob/sycl/sycl/doc/UsersManual.md, it is OK to remove this from common flags.
```
Normally, '-fsycl-targets' is specified when linking an application, in
which case the AOT compiled device binaries are embedded within the
application’s fat executable.  However, this option may also be used in
combination with '-c' and '-fno-sycl-rdc' when compiling a source file.
In this case, the AOT compiled device binaries are embedded within the fat
object file.
```